crates/compiler/builtins/bitcode/src/compiler_rt.zig

```diff
@@ -17,17 +17,24 @@ const v2u64 = @Vector(2, u64);
 
 // Export it as weak incase it is already linked in by something else.
 comptime {
-    @export(__muloti4, .{ .name = "__muloti4", .linkage = .Weak });
-    @export(__lshrti3, .{ .name = "__lshrti3", .linkage = .Weak });
+    // On Windows this gives lld-link: error: duplicate symbol: __muloti4
+    // >>> defined at compiler_rt.lib(compiler_rt.obj)
+    // lld-link: error: duplicate symbol: __lshrti3
+    // >>> defined at compiler_rt.lib(compiler_rt.obj)
+    if (builtin.os.tag != .windows) {
+        @export(__muloti4, .{ .name = "__muloti4", .linkage = .Weak });
+        @export(__lshrti3, .{ .name = "__lshrti3", .linkage = .Weak });
+    }
+
     if (want_windows_v2u64_abi) {
-        @export(__divti3_windows_x86_64, .{ .name = "__divti3", .linkage = .Weak });
-        @export(__modti3_windows_x86_64, .{ .name = "__modti3", .linkage = .Weak });
-        @export(__umodti3_windows_x86_64, .{ .name = "__umodti3", .linkage = .Weak });
-        @export(__udivti3_windows_x86_64, .{ .name = "__udivti3", .linkage = .Weak });
-        @export(__fixdfti_windows_x86_64, .{ .name = "__fixdfti", .linkage = .Weak });
-        @export(__fixsfti_windows_x86_64, .{ .name = "__fixsfti", .linkage = .Weak });
-        @export(__fixunsdfti_windows_x86_64, .{ .name = "__fixunsdfti", .linkage = .Weak });
-        @export(__fixunssfti_windows_x86_64, .{ .name = "__fixunssfti", .linkage = .Weak });
+        // @export(__divti3_windows_x86_64, .{ .name = "__divti3", .linkage = .Weak });
+        // @export(__modti3_windows_x86_64, .{ .name = "__modti3", .linkage = .Weak });
+        // @export(__umodti3_windows_x86_64, .{ .name = "__umodti3", .linkage = .Weak });
+        // @export(__udivti3_windows_x86_64, .{ .name = "__udivti3", .linkage = .Weak });
+        // @export(__fixdfti_windows_x86_64, .{ .name = "__fixdfti", .linkage = .Weak });
+        // @export(__fixsfti_windows_x86_64, .{ .name = "__fixsfti", .linkage = .Weak });
+        // @export(__fixunsdfti_windows_x86_64, .{ .name = "__fixunsdfti", .linkage = .Weak });
+        // @export(__fixunssfti_windows_x86_64, .{ .name = "__fixunssfti", .linkage = .Weak });
```

Comment:
I'm not sure why, but with these uncommented I get `lld-link: error: duplicate symbol` errors for each of these.

Reply:
let's remove the now-commented lines

Reply:
and then simplify the `if`, I guess

Reply:
Removed